lfs/lfs.go

```diff
@@ -16,10 +16,6 @@ import (
 	"github.com/rubyist/tracerx"
 )
 
-const (
-	Version = "1.5.0"
```

Comment:
Nope, `release-1.5` is basically its own thing. Bug fixes will have to be backported to it. The `master` branch is diverging a lot already, with the scanner refactoring PRs.

I suppose we should change it to `2.0.0-dev` or something though.